### PR TITLE
Project eight

### PR DESCRIPTION
**Pull request by volodymyrtaras**

### CODE REVIEW

File: src/lesson/eight/CallableClass.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package lesson.eight;
+
+import java.util.concurrent.Callable;
+
+import static java.lang.Thread.sleep;
+
+class CallableClass implements Callable<String> {
+    @Override
+    public String call() throws Exception {
```
Comment (Taniann) [inline src/lesson/eight/CallableClass.java]: better throw more specific exception InterruptedException or handle it in method with try-catch block

File: src/lesson/eight/MyThread.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package lesson.eight;
+
+import java.util.concurrent.CountDownLatch;
+
+public class MyThread implements Runnable {
+    String name;
```
Comment (Taniann) [inline src/lesson/eight/MyThread.java]: better add access modificator here

File: src/lesson/eight/CallableExample.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+
+public class CallableExample {
+
+    public CallableExample() {
```
Comment (Taniann) [inline src/lesson/eight/CallableExample.java]: it is not a good decision to realize all logic in constructor. better use for logic method main

File: src/lesson/eight/MyThread.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    CountDownLatch latch;
```
Comment (Taniann) [inline src/lesson/eight/MyThread.java]: and here

File: src/lesson/eight/Sum.java [hunk 5]
```diff
@@ -0,0 +1,33 @@
+package lesson.eight;
+
+import java.util.concurrent.RecursiveTask;
+
+public class Sum extends RecursiveTask<Long> {
+    int low;
```
Comment (Taniann) [inline src/lesson/eight/Sum.java]: add access modificator to its fields

**Comment by volodymyrtaras** [comment]
@Taniann fixed.

**Pull request merged**